apps/participants/serializers.py

```diff
@@ -102,6 +103,10 @@ def __init__(self, challenge_participant_team_list):
 class ChallengeParticipantTeamListSerializer(serializers.Serializer):
     """Serializer to map a challenge's participant team lists."""
     challenge_participant_team_list = ChallengeParticipantTeamSerializer(many=True)
+    time = serializers.SerializerMethodField()
+
+    def get_time(self, obj):
+        return timezone.now().replace(microsecond=0)
```

Comment:
Hey @Ram81 The best way would be not to change the original data, so let's keep the return statement to be `return timezone.now()`.